python/ml4ir/base/pipeline.py

```diff
@@ -296,7 +297,7 @@ def run(self):
             relevance_model = self.get_relevance_model()
             self.logger.info("Relevance Model created")
 
-            if self.args.execution_mode in {
+            '''if self.args.execution_mode in {
```

Comment:
@marjanhs Did you mean to comment this out? For testing, you could probably run in save only mode to make things faster, I guess.

Reply:
oh yes, sorry . fixed it